control-plane/subcommand/inject-connect/command.go

```diff
@@ -34,19 +35,20 @@ import (
 type Command struct {
 	UI cli.Ui
 
-	flagListen               string
-	flagCertDir              string // Directory with TLS certs for listening (PEM)
-	flagDefaultInject        bool   // True to inject by default
-	flagConsulImage          string // Docker image for Consul
-	flagEnvoyImage           string // Docker image for Envoy
-	flagConsulK8sImage       string // Docker image for consul-k8s
-	flagACLAuthMethod        string // Auth Method to use for ACLs, if enabled
-	flagWriteServiceDefaults bool   // True to enable central config injection
-	flagDefaultProtocol      string // Default protocol for use with central config
-	flagConsulCACert         string // [Deprecated] Path to CA Certificate to use when communicating with Consul clients
-	flagEnvoyExtraArgs       string // Extra envoy args when starting envoy
-	flagLogLevel             string
-	flagLogJSON              bool
+	flagListen                string
+	flagCertDir               string // Directory with TLS certs for listening (PEM)
+	flagDefaultInject         bool   // True to inject by default
+	flagConsulImage           string // Docker image for Consul
+	flagEnvoyImage            string // Docker image for Envoy
+	flagConsulK8sImage        string // Docker image for consul-k8s
+	flagACLAuthMethod         string // Auth Method to use for ACLs, if enabled
+	flagWriteServiceDefaults  bool   // True to enable central config injection
+	flagDefaultProtocol       string // Default protocol for use with central config
+	flagConsulCACert          string // [Deprecated] Path to CA Certificate to use when communicating with Consul clients
+	flagEnvoyExtraArgs        string // Extra envoy args when starting envoy
+	flagEnableWebhookCAUpdate bool
```

Comment:
- Expose this `enable-webhook-ca-update` which basically says "webhook-cert-manager is turned off so we are going to configure webhook certs directly here
- it will use common code in `mutatingwebhookconfiguration.UpdateWithCABundle()` and will use the `ca.crt` in the `flagCertDir` directory